Gemfile

```
@@ -190,7 +190,7 @@ gem 'highline', '~> 3.1.0'
 
 gem 'honeybadger', '>= 4.5.6' # error monitoring
 
-gem 'newrelic_rpm', '~> 8.3', group: [:staging, :development, :production] # perf/error/etc monitoring
+gem 'newrelic_rpm', '~> 8.3', require: false # perf/error/etc monitoring
```

why do you want to make this available in levelbuilder, test and adhoc environments? If you do not explicitly need it there (you don't mention it in the PR description) then why not:
```suggestion
gem 'newrelic_rpm', '~> 8.3', group: [:staging, :development, :production], require: false # perf/error/etc monitoring
```

---

We do not — we simply avoid requiring it anywhere by default, allowing the agent's loading to be controlled by the `CDO.newrelic_logging` config, as was originally intended:
https://github.com/code-dot-org/code-dot-org/blob/0967ea4e6018ce6d6af59405686974d30b2854a0/dashboard/config/application.rb#L236-L238

---

👍

---

To clarify: this change has the effect both of no longer requiring the gem by default AND of installing it in environments it was not being installed in before

---

good point -- I agree with Elijah that we shouldn't start installing this gem in new environments, so I now i like my suggested change above of adding `group: [...]` back in again.